packages/config/src/index.ts

```diff
@@ -1,42 +1,35 @@
-import {GENESIS_SLOT, IBeaconParams} from "@chainsafe/lodestar-params";
-import {createIBeaconSSZTypes, IAllForksSSZTypes, Slot, Version} from "@chainsafe/lodestar-types";
-
-import {IBeaconConfig, IForkInfo, ForkName} from "./interface";
+import {GENESIS_EPOCH, IBeaconParams} from "@chainsafe/lodestar-params";
+import {createIBeaconSSZTypes, Slot, IAllForksSSZTypes, Version} from "@chainsafe/lodestar-types";
+import {IBeaconConfig, ForkName} from "./interface";
 
 export * from "./interface";
 
 export function createIBeaconConfig(params: IBeaconParams): IBeaconConfig {
   const types = createIBeaconSSZTypes(params);
+
+  const phase0 = {name: ForkName.phase0, epoch: GENESIS_EPOCH, version: params.GENESIS_FORK_VERSION};
+  const altair = {name: ForkName.altair, epoch: params.ALTAIR_FORK_EPOCH, version: params.ALTAIR_FORK_VERSION};
+
   return {
     params,
     types,
-    getForkInfoRecord(): Record<ForkName, IForkInfo> {
-      return {
-        phase0: {
-          name: ForkName.phase0,
-          slot: GENESIS_SLOT,
-          version: params.GENESIS_FORK_VERSION,
-        },
-        altair: {
-          name: ForkName.altair,
-          slot: params.ALTAIR_FORK_SLOT,
-          version: params.ALTAIR_FORK_VERSION,
-        },
-      };
+
+    forks: {altair, phase0},
```

Comment:
We need / I'd like a place that has the `IForkInfo` objects _in order of occurance_. (I've been using `getForkInfoRecord` for that purpose in a local branch)